pkg/util/restoptions/hacks.go

```diff
@@ -0,0 +1,79 @@
+package restoptions
+
+import (
+	genericrest "k8s.io/kubernetes/pkg/registry/generic"
+	"k8s.io/kubernetes/pkg/runtime"
+	"k8s.io/kubernetes/pkg/storage"
+	etcdstorage "k8s.io/kubernetes/pkg/storage/etcd"
+	"k8s.io/kubernetes/pkg/storage/storagebackend"
+	"k8s.io/kubernetes/pkg/storage/storagebackend/factory"
+)
+
+// Temporary hack until rebase of upstream PR 37770 ///////////////////////////////////////////////
```

Comment:
Wait until then for the entire PR / trello card?  I suppose that depends on how quickly we want these features.